static/sass/styles.scss

```diff
@@ -294,9 +294,35 @@ html {
   display: inline-block;
 }
 
-.middle-page-tabs {
-  @media screen and (min-width: $breakpoint-large) {
-    padding-left: 20.3rem;
+.p-tabs--25-75 {
+  @extend %vf-row;
+  position: relative;
+  margin-bottom: 1.5rem;
+
+  &::before {
+    content: '';
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    height: 1px;
+    width: 100%;
+    background-color: $colors--theme--border-default;
+  }
+
+  .p-tabs__list {
+    @extend %vf-pseudo-border--bottom;
+    margin-bottom: 0;
+    padding-left: 0.5rem;
```

Comment:
This will have to be adjusted (to be negative margin) once the top navigation is fixed to use `p-navigation__row--25-75`.